redisinsight/ui/src/components/triggers/insights-trigger/insights-trigger.styles.ts

```diff
@@ -0,0 +1,40 @@
+import styled, { css } from "styled-components"
+
+import { IconButton } from "uiSrc/components/base/forms/buttons"
+import { IconButtonProps } from "uiSrc/components/base/forms/buttons/IconButton"
+
+export const BulbWrapper = styled.div`
+  position: relative;
+`
+
+export const BulbHighlighting = styled.span`
+  // TODO: Using the background color from the previous value until there is an appropriate color 
+  // from the pallete to use for both light and dark themes.
+  background-color: #ffaf2b;
```

Comment:
We already have such a color defined in the palette, so you can use it.

```suggestion
  background-color: ${({ theme }) => theme.color.orange500};
```